mmseg/core/evaluation/metrics.py

```diff
@@ -21,25 +22,29 @@ def intersect_and_union(pred_label,
             work only when label is str. Default: False.
 
      Returns:
-         ndarray: The intersection of prediction and ground truth histogram
-             on all classes.
-         ndarray: The union of prediction and ground truth histogram on all
-             classes.
-         ndarray: The prediction histogram on all classes.
-         ndarray: The ground truth histogram on all classes.
+         torch.Tensor: The intersection of prediction and ground truth
+         histogram on all classes.
+         torch.Tensor: The union of prediction and ground truth histogram on
+         all classes.
+         torch.Tensor: The prediction histogram on all classes.
+         torch.Tensor: The ground truth histogram on all classes.
     """
 
     if isinstance(pred_label, str):
-        pred_label = np.load(pred_label)
+        pred_label = torch.from_numpy(np.load(pred_label))
+    else:
+        pred_label = torch.from_numpy((pred_label))
 
     if isinstance(label, str):
-        label = mmcv.imread(label, flag='unchanged', backend='pillow')
-    # modify if custom classes
+        label = torch.from_numpy(
+            mmcv.imread(label, flag='unchanged', backend='pillow'))
+    else:
+        label = torch.from_numpy(label)
```

Comment:
We may add test function for this part. 
You may create some tempfile to achieve thi.s